src/com/elikill58/negativity/universal/Cheat.java

```diff
@@ -167,4 +171,30 @@ public static void loadCheat() {
 	public static enum CheatCategory {
 		COMBAT, MOVEMENT, WORLD, PLAYER;
 	}
+	
+	public static class CheatHover {
+		private final String key;
+		private final Object[] placeholders;
+		
+		public CheatHover(String key, Object... placeholders) {
+			this.key = key;
+			this.placeholders = placeholders;
+		}
+
+		public String getKey() {
+			return key;
+		}
+
+		public Object[] getPlaceholders() {
+			return placeholders;
+		}
+		
+		public String compile() {
+			return key == null ? "" : "\n" + TranslatedMessages.getStringFromLang(TranslatedMessages.DEFAULT_LANG, getKey(), getPlaceholders());
+		}
+		
+		public String compile(NegativityPlayer np) {
+			return key == null ? "" : "\n" + TranslatedMessages.getStringFromLang(np.getAccount().getLang(), getKey(), getPlaceholders());
+		}
```

Comment:
```suggestion
		public String compile() {
			return "\n" + TranslatedMessages.getStringFromLang(TranslatedMessages.DEFAULT_LANG, getKey(), getPlaceholders());
		}
		
		public String compile(NegativityPlayer np) {
			return "\n" + TranslatedMessages.getStringFromLang(np.getAccount().getLang(), getKey(), getPlaceholders());
		}
```
D'ailleurs je trouve que préfixer `"\n"` très _meh_, ça n'a pas sa place ici, même si ça fonctionne bien pour l'instant, là où ces méthodes sont appelées.